### ISSUE TITLE
A little mistake in pix2pix and pix2pixHD code

### ISSUE THREAD

**Issue by GiantKevin** [SerialLain3170/Colorization#9]
https://github.com/SerialLain3170/Colorization/blob/66915f3875720f6223a51efaee358f3c3e315bef/nohint_pix2pixHD/model.py#L117

`if norm == "bn":
            modules.append(nn.BatchNorm2d(out_ch))
elif norm == "in":
            modules.append(**nn.BatchNorm2d(out_ch)**)
`
**nn.BatchNorm2d(out_ch)** should be **nn.InstanceNorm2d(out_ch)**.

Anyway, this project is great, thanks for your sharing.

**Comment by SerialLain3170** [comment]
Thanks, I had not noticed the bug. I'll fix it.
